src/common/TrackedOp.h

```diff
@@ -151,6 +152,8 @@ class TrackedOp : public boost::intrusive::list_base_hook<> {
   std::atomic_int nref = {0};  ///< ref count
 
   utime_t initiated_at;
+  PG* pg_ptr;
+  bool pglock_can_release_flag;
```

Comment:
This would all need to go in the OSD-specific OpRequest implementation rather than the common code here that is shared by the mds and monitor.